Modules/System/SharePoint/src/model/listitemattachment/SharePointListItemAtch.Table.al

```diff
@@ -0,0 +1,71 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+/// <summary>
+/// Holds information about list item attachment entity.
+/// </summary>
+table 9104 "SharePoint List Item Atch"
+{
+    Access = Public;
+    DataClassification = SystemMetadata;
+    Caption = 'SharePoint List Item Attachment';
+    TableType = Temporary;
+    Extensible = false;
+
+    fields
+    {
+
+        field(1; OdataId; Text[2048])
```

Comment:
Do the names have to `OdataId`? Or can it be `"Odata Id"`. There is no consistency in the naming as the Odata ones are all concatenated while others are separated with "".

Reply:
Hi,

All the field names are 1:1 with naming used by SharePoint.
At some point, I would like to introduce the ability for users to build OData queries to retrieve the exact data they need.
If field names are consistent with SharePoint then it will be easier to build filters, otherwise, we will need a mapping.

Maybe we can keep original field names but normalize captions?

Reply:
Hi, as agreed we will keep the SharePoint naming.